cypress/support/fragments/circulation/newNoticePolicyTemplate.js

```diff
@@ -134,5 +138,20 @@ export default {
       Button({ id: 'dropdown-clickable-delete-item' }).click(),
       Button({ id: 'clickable-delete-item-confirmation-confirm' }).click(),
     ]);
+  },
+
+  duplicateTemplate: () => {
+    cy.do([
+      actionsButton.click(),
+      actionsButtons.duplicate.click(),
+    ]);
+  },
+
+  typeTemplateName: (noticePolicytemplateName) => {
+    cy.get('#input-patron-notice-name').clear().type(noticePolicytemplateName);
```

Comment:
actually we need to use here an interactor TextField, but I am not 100% sure that it works properly in this case

Reply:
done